.github/workflows/e2e-dev.yml

```diff
@@ -12,15 +12,19 @@ jobs:
         browser: ['chrome', 'firefox']
     steps:
       - uses: actions/checkout@v2
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 7.4.0
+          run_install: true
       - name: Cypress run
         uses: cypress-io/github-action@v2
         with:
-          install-command: yarn install
+          install: false
```

Comment:
cc @logan-anderson just FYI the cypress image doesn't know about pnpm, so we do the install ourselves.